### PR TITLE
feat: support creating and deleting users

### PR DESCRIPTION
**Pull request by circa10a**
This PR supports creating a deleting of Jenkins users/accounts.

### CODE REVIEW

File: users.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+		}
+	}
+	return user, nil
+}
+
+// DeleteUser deletes a Jenkins account
+func (j *Jenkins) DeleteUser(ctx context.Context, userName string) error {
+	deleteContext := "/securityRealm/user/" + userName + "/doDelete"
+	payload := "Submit=Yes"
+	fmt.Println(j.Version)
```
Comment (figo) [inline users.go:57]: I believe this is for debugging purpose, could you remove this line?

**Comment by figo** [comment]
No description in this PR, could you add proper description, thanks

File: users.go (hunk 1 above)
Comment (circa10a) [reply users.go]: Fixed. Sorry I missed this

**Comment by circa10a** [comment]
@figo Fixed

**Pull request merged**